anago

```diff
@@ -650,6 +646,9 @@ get_build_candidate () {
     # If the branch is a 3-part branch (ie. release-1.2.3)
     if [[ $RELEASE_BRANCH =~ $BRANCH_REGEX ]] && \
        [[ -n ${BASH_REMATCH[4]} ]]; then
+      ((FLAGS_official)) \
```

Comment:
This is actually a 2 part process described in [docs/branching.md](https://github.com/kubernetes/release/blob/master/docs/branching.md#branching-from-a-tag).
This is a necessary step due to the way "latest" tags are consumed by things like GKE.  Once a tag is made, there are bits expected to be there.  Otherwise, the branching/tagging step would be much simpler (and faster).  

```
# Branch from tag and create all the builds and artifacts required by GKE
$ anago release-1.6.3
# Create the next release on that new branch
$ anago release-1.6.3 --official
# And then add --nomock to both above to do the real thing
```